src/LexicographicHelpers.dfy

```diff
@@ -0,0 +1,243 @@
+// RUN: %dafny /compile:0 "%s"
+
+/*******************************************************************************
+*  Copyright by the contributors to the Dafny Project
+*  SPDX-License-Identifier: MIT 
+*******************************************************************************/
+
+include "Relations.dfy"
+include "Lexicographics.dfy"
+include "UInt.dfy"
+
+module LexicographicHelpers{
```

Comment:
Could you please rename this `LexicographicSequence`, so that you can remove all the "Lex" prefix from all the procedures below?